app/templates/game.html.twig

```diff
@@ -2,30 +2,23 @@
 
 {% block header %}   
   <div class="section">
-    <h5>Current game</h5>
+    {% if isVictory %}
+      <h5>Congratulations, you have solved the puzzle !</h5>
```

Comment:
Can you use translation system ?
http://symfony.com/doc/master/translation.html

Reply:
Unfortunately I'll not have the time to apply this review. I'll keep that for another PR.